src/components/event-details/EventDetails.tsx

```diff
@@ -106,10 +118,33 @@ const EventListing: React.FC<EventProps> = ({ thumbnail, title, details }) => {
           <GreenButton>Register</GreenButton>
         </Col>
         <Col span={10}>
-          <Title level={5}>Announcements</Title>
-          <AnnouncementBox>
-            There are no announcements for this event
-          </AnnouncementBox>
+          {announcements && (
+            <>
+              {announcements.length ? (
```

Comment:
❓ Why is there a check for both announcements not being undefined and having a non-zero length? Is it possible for [] to be passed in as a prop? If so, it seems like the prop shouldn't be optional

Reply:
Need to check if the announcements prop was passed in (because its optional), and then there's two different displays for = 0 and > 0 length